web/modules/custom/nlc_register_interest/templates/nlc-register-interest-header.html.twig

```diff
@@ -2,6 +2,6 @@
 <h2 class="govuk-heading-m"><svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="bell" class="svg-inline--fa fa-bell fa-w-14" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512">
 <path fill="currentColor" d="M224 512c35.32 0 63.97-28.65 63.97-64H160.03c0 35.35 28.65 64 63.97 64zm215.39-149.71c-19.32-20.76-55.47-51.99-55.47-154.29 0-77.7-54.48-139.9-127.94-155.16V32c0-17.67-14.32-32-31.98-32s-31.98 14.33-31.98 32v20.84C118.56 68.1 64.08 130.3 64.08 208c0 102.3-36.15 133.53-55.47 154.29-6 6.45-8.66 14.16-8.61 21.71.11 16.4 12.98 32 32.1 32h383.8c19.12 0 32-15.6 32.1-32 .05-7.55-2.61-15.27-8.61-21.71z"></path></svg>
 &nbsp;You now have the opportunity to take part in the NLC Leadership Programme.</h2>
-<p><a href='#nlc-register-interest-show-more-button' role="button" aria-controls="nlc-register-interest-show-more-content" id="nlc-register-interest-show-more">Show more</a></p>
+<p><a href='#nlc-register-interest-show-more-button' role="button" aria-controls="nlc-register-interest-show-more-content" aria-expanded="false" id="nlc-register-interest-show-more">Show more</a></p>
```

Comment:
So, just to check with what you said in your message in [ch1795] — this sets a default state that's only visible in mobile. Does it need to change when the thing indicated in `aria-controls` is expanded? Or is it fine that it just stays like this? I'm not really familiar with the needs. Maybe @SamMed1 has an insight?

Reply:
Yep the previous commits in this branch changes the aria-expanded attribute dynamically. This is just adding one by default.

Reply:
Ace, thanks.